examples/python/blueprint_stocks/main.py

```diff
@@ -0,0 +1,215 @@
+            )
+            for symbol in symbols
+        ]
+    )
+
+
+def hide_panels(viewport: rrb.ViewportLike) -> rrb.BlueprintLike:
+    """Wrap a viewport in a blueprint that hides the time and selection panels."""
+    return rrb.Blueprint(
+        viewport,
+        rrb.TimePanel(expanded=False),
+        rrb.SelectionPanel(expanded=False),
+    )
+
+
+################################################################################
+# Helper functions for styling
+################################################################################
+
+brand_colors = {
+    "AAPL": 0xA2AAADFF,
+    "AMZN": 0xFF9900FF,
+    "GOOGL": 0x34A853FF,
+    "META": 0x0081FBFF,
+    "MSFT": 0xF14F21FF,
+}
+
+
+def style_plot(symbol: str) -> rr.SeriesLine:
+    return rr.SeriesLine(
+        color=brand_colors[symbol],
+        name=symbol,
+    )
+
+
+def style_peak(symbol: str) -> rr.SeriesPoint:
+    return rr.SeriesPoint(
+        color=0xFF0000FF,
+        name=f"{symbol} (peak)",
+        marker="Up",
+    )
+
+
+################################################################################
+# Main script
+################################################################################
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Visualize stock data using the Rerun SDK")
+    parser.add_argument(
+        "--blueprint",
+        choices=["auto", "one-stock", "one-stock-with-info", "compare-two", "one-stock-no-peaks", "grid"],
+        default="grid",
+        help="Select the blueprint to use",
+    )
+    parser.add_argument(
+        "--show_panels",
+        action="store_true",
+        help="Show the time and selection panels",
+    )
+
+    rr.script_add_args(parser)
+    args = parser.parse_args()
+
+    et_timezone = pytz.timezone("America/New_York")
+    current_date = dt.datetime.now(et_timezone).date()
+    symbols = ["AAPL", "AMZN", "GOOGL", "META", "MSFT"]
+    dates = list(filter(lambda x: x.weekday() < 5, [current_date - dt.timedelta(days=i) for i in range(7, 0, -1)]))
+
+    blueprint: rrb.BlueprintLike
+
+    if args.blueprint == "auto":
+        blueprint = auto_blueprint()
+    elif args.blueprint == "one-stock":
+        blueprint = one_stock("AAPL")
+    elif args.blueprint == "one-stock-with-info":
+        blueprint = one_stock_with_info("AMZN")
+    elif args.blueprint == "one-stock-no-peaks":
+        blueprint = one_stock_no_peaks("GOOGL")
+    elif args.blueprint == "compare-two":
+        blueprint = compare_two("META", "MSFT", dates[-1])
+    elif args.blueprint == "grid":
+        blueprint = stock_grid(symbols, dates)
+    else:
+        raise ValueError(f"Unknown blueprint: {args.blueprint}")
+
+    if not args.show_panels:
+        blueprint = hide_panels(blueprint)
+
+    rr.init("rerun_example_stocks", spawn=True, blueprint=blueprint)
+
+    # In a future blueprint release, this will move into the blueprint as well
```

Comment:
```suggestion
    # In a future blueprint release, this can move into the blueprint as well
```

I think it's important to point out that this isn't becoming entirely obsolete 🤷